ptsprojects/zephyr/sm_wid.py

```diff
@@ -16,6 +16,7 @@
 import logging
 import sys
 import re
+import time
```

Comment:
If we don't call `time.sleep` we won't need this one

Reply:
fixed